docker/unlock.tests.dockerfile

```diff
@@ -4,4 +4,5 @@ RUN mkdir /home/unlock
 COPY . /home/unlock
 WORKDIR /home/unlock
 
+ENV NODE_ENV=test
```

Comment:
Same, did tests no pass before that?

Reply:
I think our travis setup is [currently](https://github.com/unlock-protocol/unlock/issues/195) 💩... thanks!